.github/workflows/fluxctl-snap.yaml

```diff
@@ -0,0 +1,31 @@
+name: Build snap
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-snap:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Install Snapcraft
+        uses: samuelmeuli/action-snapcraft@v1
+        with:
+          snapcraft_token: ${{ secrets.snapcraft_token }}
+          use_lxd: true
+
+      - name: Run Snapcraft build
+        run: sudo snapcraft --use-lxd
+
+      - name: Upload to Snapcraft store (edge channel for revs in master)
+        if: ${{ ! startsWith(github.event.ref, 'refs/tags') }}
+        run: snapcraft push --release edge fluxctl_*.snap
+
+      - name: Upload to Snapcraft store (stable channel for tagged releases)
+        if: startsWith(github.event.ref, 'refs/tags')
```

Comment:
How come this doesn't get mustaches around it, like the stanza above?

Reply:
The `!` above wanted to be evaluated I found. :shrug:

Reply:
How strange, so the negation of an expression is _not_ an expression ...

Reply:
I'm happy to add moustaches. Let me try.

Reply:
I was thinking maybe the one in mustaches didn't need them

Reply:
No, that I tried out - it failed.